utils/coreutils/techutils.go

```diff
@@ -37,6 +37,33 @@ const (
 
 const Pypi = "pypi"
 
+func TechnologyToLanguage(technology Technology) CodeLanguage {
+	languageMap := map[Technology]CodeLanguage{
+		Npm:    JavaScript,
+		Pip:    Python,
+		Poetry: Python,
+		Pipenv: Python,
+		Go:     GoLang,
+		Maven:  Java,
+		Gradle: Java,
+		Nuget:  CSharp,
+		Dotnet: CSharp,
+		Yarn:   JavaScript,
+		Pnpm:   JavaScript,
+	}
+	return languageMap[technology]
+}
```

Comment:
Instead of this func, you can attach the value directly to the `TechData` as new attribute `Language CodeLanguage`

Reply:
I don't understand your comment, but what is wrong with this implementation?